src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -138,19 +126,34 @@ public static final class Roller {
 
         //#endregion
 
-
         //#region Ports
         public static final int rollerPort = 18;
         public static final int ledPort = 0;
 
         //#endregion
 
-
-
         //#region Command Constants
-
-        public static final double coneIntakeConeOuttakeSpeed = -.3;
-        public static final double coneOuttakeConeIntakeSpeed = .7;
+        // TODO: Determine actual speeds/timings for roller
+        public static class RollerMode {
+            public static RollerMode INTAKE_CONE = new RollerMode(-0.5, .5, true);
+            public static RollerMode INTAKE_CUBE = new RollerMode(0.3, .25, true);
+            public static RollerMode OUTTAKE_CONE = new RollerMode(0.5, .5, false);
+            public static RollerMode OUTTAKE_CUBE = new RollerMode(-0.5, .5, false);
```

Comment:
Make `final`

Reply:
Those values have not been fully determined yet. I was planning at some point to do a test with the rollers to determine the timings of the cube/cone.